packages/bbot/data_stream/asm_intel/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,114 @@
+---
+description: Pipeline for ingesting BBOT ndjson files.
+
+processors:
+####################
+# Event ECS Fields #
+####################
+- set:
+    field: ecs.version
+    value: 8.12.0
+- set:
+    field: event.kind
+    value: asset
+- json:
+    field: message
+    target_field: bbot
+    tag: json_message
+- date:
+    field: bbot.timestamp
+    formats:
+    -   UNIX
+    if: ctx?.bbot?.timestamp != null
+    ignore_failure: false
+##############
+# ECS Fields #
+##############
+- rename:
+    field: bbot.data.PROTOCOL.port
+    target_field: url.port
+    ignore_missing: true
+- rename:
+    field: bbot.data.VULNERABILITY.severity
+    target_field: vulnerability.severity
+    ignore_missing: true
+- rename:
+    field: bbot.data.DNS_NAME
+    target_field: host.name
+    ignore_missing: true
+- rename:
+    field: bbot.resolved_hosts
+    target_field: related.hosts
+    ignore_missing: true
+- append:
+    field: host.ip
+    value: ["{{bbot.data.IP_ADDRESS}}"]
```

Why is this wrapped in brackets?
```suggestion
    value: '{{{bbot.data.IP_ADDRESS}}}'
```
(similar throughout)